src/App.jsx

```diff
@@ -1,104 +1,28 @@
 import './App.scss';
+import { PostList } from './components/PostList/PostList';
```

Comment:
Ensure that the styles for the `App` component are moved from `App.scss` to a separate `.scss` file for the `App` component as per the task requirements.